cypress/integration/features.spec.js

```diff
@@ -0,0 +1,184 @@
+beforeEach(() => {
+  cy.visit('http://localhost:3000')
+})
+
+describe('Test if demo app running', () => {
+  it('Visits localhost website', () => {
+    cy.visit('http://localhost:3000')
+  })
+})
+
+describe('Check if FormContextWithValidation works', () => {
```

Comment:
May we have a describe for file and a single test for components? I would suggest to follow the : https://github.com/iusehooks/usetheform/tree/master/__tests__ structure or similar and also to create a new index.html page named cypress.html or better to add a new script which runs a web server like the one running with the command npm run dev and make it to point to cypress folder:

```
    "cypress:server": "rimraf dev coverage && concurrently \"node ./build_config/dev.js\" \"browser-sync start --server cypress\server --ss dev --files dev --files cypress\server\""

```